website/src/pages/cwkw/cwkw-layout.tsx

```diff
@@ -47,7 +47,7 @@ const CWKWLayout: React.FC = ({ children }) => {
           }
         >
           {isDesktop ? <Sidebar /> : <MobileSidebar />}
-          <div className={css.contentContainer}>
+          <span className={css.contentContainer}>
             <h1 className={css.siteTitle}>
```

Comment:
This is unintuitive to me. A `span` with a `h1` and `p` in it is sort of odd, no? Also, we just make it `display: block` again anyways. If this is valid HTML and there is a reason behind it, I think it warrants a comment.

Reply:
oops, that was just me getting my HTML semantics scrambled. Will fix!